Source/WebCore/platform/graphics/MediaPlayerEnums.h

```diff
@@ -110,6 +111,13 @@ enum class MediaPlayerNeedsRenderingModeChanged : bool {
     Yes,
 };
 
+enum class MediaPlayerVideoPlaybackConfigurationOption : uint8_t {
+    Stereo = 1 << 0,
+    StereoMultiview = 1 << 1,
+    Spatial = 1 << 2,
+};
```

Comment:
Should we have a Mono here? And have the concept of an empty set mean "I don't know yet"?

Reply:
I had a "Default" at the beginning, but the AVF API doesn't really tell you that. Also I was a bit wondering of what should be done when we don't have a video track yet: should be "nothing" or "mono" ?

I think the later is better, will do